src/components/Table/TableRow/TableRow.scss

```diff
@@ -9,36 +9,40 @@ $row-multiple-line-height: 64px;
     .moonstone-TableBody &:hover {
```

Comment:
Also don't see the relation with icons here ?

Reply:
Finally, I kept the nested classes and just added this scss snippet.